daprdocs/content/en/contributing/daprbot.md

```diff
@@ -0,0 +1,29 @@
+---
+type: docs
+title: "Dapr bot reference"
+linkTitle: "Dapr bot"
+weight: 15
+description: "List of all Dapr bot capabilities."
+---
+
+Dapr bot is a GitHub Script that helps with common tasks in the Dapr organization. It is set up individually for each repository ([example](https://github.com/dapr/dapr/blob/master/.github/workflows/dapr-bot.yml)) and can be configured to run on specific events. This reference covers the dapr bot capabilities from the dapr and components-contrib repositories only.
+
+## command reference
+
+| Command | Target | Description | Who can use | Repository |
+|---------|--------|-------------|-------------|------------|
+| `/assign` | Issue | Assigns an issue to a user or group of users | Anyone | dapr, components-contrib |
+| `/ok-to-test` | Pull request | dapr: trigger end to end tests <br/> components-contrib: trigger conformance and certification tests | Users listed in the [bot](https://github.com/dapr/dapr/blob/master/.github/scripts/dapr_bot.js)  | dapr, components-contrib |
+| `/ok-to-perf` | Pull request | Trigger performance tests. | Users listed in the [bot](https://github.com/dapr/dapr/blob/master/.github/scripts/dapr_bot.js) | dapr |
+| `/make-me-laugh` | Issue or pull request | Posts a random joke | Users listed in the [bot](https://github.com/dapr/dapr/blob/master/.github/scripts/dapr_bot.js) | dapr, components-contrib |
+
+## label reference
+
+You can query issues created by the dapr-bot by using the `created-by/dapr-bot` label.
```

Comment:
@shubham1172 what do you mean by this? Can you clarify or show an example of this?

Reply:
Should I remove this line for now, or add this link (that currently does not point to any issues)?

Reply:
When will dapr bot start to create issues? If soon, I think we can use that link and add it in. Otherwise, I'd remove it for now

Reply:
Yes, it is already live. Let me add a link here.